.travis.yml

```diff
@@ -1,9 +1,8 @@
 language: java
 
 jdk:
-  - oraclejdk8
-  - oraclejdk9
-  - oraclejdk10
+  - openjdk8
+  - openjdk9
   - openjdk10
   - openjdk11
-  - openjdk-ea
+  - openjdk13
```

Comment:
Add `openjdk12` as well?

Reply:
I thought it's not relevant anymore now, but will add anyway